components/brave_rewards/resources/ui/stories/wallet.tsx

```diff
@@ -72,19 +72,19 @@ storiesOf('Rewards/Wallet/Desktop', module)
         ]}
         grants={showGrant ? [
           {
-            tokens: '8.0',
-            expireDate: '7/15/2018',
-            type: 'ugp'
+            amount: 2.5,
+            expiresAt: '1574451334789',
+            type: 1
```

Comment:
I think this is an aside, but could / do we have a enum-like thing for this type value. For example GrantType.SOMETHING? (Which would evaluate to 1.